### DISCUSSION TITLE
Possibly a dumb question but : how do I play a video ?

### DISCUSSION THREAD

**Discussion by grondilu** [beakerbrowser/beaker#1846] [Q&A] [answered]
Hi.

I just wanted to test basic video file playback from a html page : 

hyper://4fa97b45c8e3ed6a779770cec36c66a7fdf97c6c2ef134628139b76cbb2f9f98/

On this drive I test audio, video and image.  Audio and image works, but I get no playback from video unless I convert the video to a data uri and insert it straight into the html file.

What am I missing ?

**Comment by pfrazee** [comment, marked as answer]
This is a bug that gets semi-fixed in the next release of Beaker. (It's a more complex issue than I would've liked!)

In 1.0, the videos will play but, tragically, seeking is broken. Still working on it 😐
